README.md

```diff
@@ -1,10 +1,10 @@
 # esi-leap
 
-esi-leap is an OpenStack service for acting as a FLOCX provider to communicate with a FLOCX
-marketplace. It is intended to work on top of the
-[esi-common library](https://github.com/CCI-MOC/esi-common), which supports an OpenStack
-install that simulates Ironic multi-tenancy through the use of `project_owner_id` and
-`project_id` property attributes.
+esi-leap is an OpenStack service for acting as a FLOCX provider to communicate
+with a FLOCX marketplace. It is intended to work on top of the
+[esi-common](https://github.com/CCI-MOC/esi-common) library, which supports an
+OpenStack install that simulates Ironic multi-tenancy through the use of
+`project_owner_id` and `project_id` property attributes.
```

Comment:
While you're here, could you update this text to remove mentions of FLOCX and esi-common? At this point, esi-leap is really a leasing service that runs on top of multi-tenant Ironic.